src/vm.rb

```diff
@@ -3,70 +3,66 @@
 require 'config'
 require 'libvirt'
 require 'metal_log'
+require 'namespaces/alces'
 
 module Metalware
   class Vm
-    def initialize(libvirt_host, node, *args)
-      @libvirt ||= Libvirt.open("qemu://#{libvirt_host}/system")
+    attr_reader :node
```

Comment:
Doesn't look like you're using this AFAICS, looks like you always access the instance variable directly - `s/@node/node/g` should do it.